loomi/regulariza-diagonal-client
# Modals

## Modals to add new areas, stamps and sections

---

- Add a new dependency react-modal
- Create a new modal component
- Add new fake post request to Nucleus and Residents APIs

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![PR_ModalsToNewArea](https://user-images.githubusercontent.com/45184938/124514247-f50b2e80-ddb2-11eb-9f39-73022875c4b9.png)
